frontend/src/metabase/databases/components/DatabaseConnectionUri/engines-config.ts

```diff
@@ -0,0 +1,116 @@
+import type { EngineKey } from "metabase-types/api/settings";
+
+export const enginesConfig = new Map<EngineKey, { placeholder: string }>([
+  [
+    "athena",
```

Comment:
why do we have multiple keys for some of these? like "athena" and "amazon-athena" and `postgres` vs `postgresql` ?